tools/dependencies.props

```diff
@@ -16,8 +16,5 @@
     <SnVersion>1.0.0</SnVersion>
     <VSWhereVersion>2.5.2</VSWhereVersion>
     <MicroBuildCoreVersion>0.2.0</MicroBuildCoreVersion>
-    <MicrosoftDotNetSignToolVersion>1.0.0-beta.19058.5</MicrosoftDotNetSignToolVersion>
```

Comment:
Note that in this file you have a bunch of versions for corefx packages that should probably instead be comminng from an arcade subscription in order to guarantee that corefxlab always has the latest version of those dependencies. You don't have to fix that now, but might be something to think about in the future.

Reply:
Good point. Testing with all of these removed.

Reply:
Actually those need to stay like that. @ahsonkhan left them like that in purpose, we don't want a subscription for those just yet, right @ahsonkhan?

Also, this file should be eng/Versions.props no? That's imported by default by arcade.

Reply:
Yup, it needed all those version numbers. I did move it to eng/Versions.props though.